### ISSUE TITLE
Broken Example - Mapchart with tooltip

### ISSUE THREAD

**Issue by morlynco** [zcreativelabs/react-simple-maps#338]
https://www.react-simple-maps.io/examples/map-chart-with-tooltip/

![image](https://github.com/zcreativelabs/react-simple-maps/assets/105128187/193354c9-5569-4c86-92d1-3136b34181ef)

**Comment by berezovskyicom** [comment]
Hi, here's a [link to a fixed version](https://codesandbox.io/p/sandbox/map-chart-with-tooltip-xwijz?file=%2Fsrc%2Findex.js%3A3%2C27&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clu1hrm8r00063b6lznvfahox%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clu1hrm8r00023b6l1r6ix6tk%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clu1hrm8r00033b6l4pwzuo5t%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clu1hrm8r00053b6ljgnhqwiw%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clu1hrm8r00023b6l1r6ix6tk%2522%253A%257B%2522id%2522%253A%2522clu1hrm8r00023b6l1r6ix6tk%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clu1hskig004y3b6j4wqb177m%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A3%252C%2522startColumn%2522%253A27%252C%2522endLineNumber%2522%253A3%252C%2522endColumn%2522%253A27%257D%255D%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.js%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clu1hskig004y3b6j4wqb177m%2522%257D%252C%2522clu1hrm8r00053b6ljgnhqwiw%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clu1hrm8r00043b6lfdf8ahw0%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clu1hrm8r00053b6ljgnhqwiw%2522%252C%2522activeTabId%2522%253A%2522clu1hrm8r00043b6lfdf8ahw0%2522%257D%252C%2522clu1hrm8r00033b6l4pwzuo5t%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clu1hrm8r00033b6l4pwzuo5t%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D).
Basically latest version of react-tooltip ([v5](https://react-tooltip.com/docs/getting-started#using-anchor-data-attributes) at the date of writing) has changed the way of importing a component.
Something to add personally - use `data-tooltip-float={true}` so that the tooltip would follow the mouse position when it moves.

**Comment by berezovskyicom** [comment]
I could also update docs but I cannot find a repo for website, probably it's private.
If it's not, would appreciate if someone could send me a link so I could make a pr with this fix.
